tests/installation/bootloader_zkvm.pm

```diff
@@ -18,44 +18,64 @@ use testapi;
 use strict;
 use warnings;
 
+
 sub run() {
 
     my $self = shift;
 
-    my $svirt = select_console('svirt');
-    my $name  = $svirt->name;
+    my $svirt    = select_console('svirt');
+    my $name     = $svirt->name;
+    my $img_path = "/var/lib/libvirt/images";
 
     # temporary use of hardcoded '+4' to workaround messed up network setup on z/KVM
     my $vtap = $svirt->instance + 4;
+    my $repo = "ftp://openqa.suse.de/" . get_var('REPO_0');
```

Comment:
ok, it's only moved here so fine for this PR but
@Soulofdestiny please replace the hardcoded values in all test files accordingly. I am awaiting your PR then :-)